README.md

```diff
@@ -10,3 +10,49 @@ Contains the SANSA Examples for [Apache Spark](http://spark.apache.org/).
 
 ### [sansa-examples-flink](https://github.com/SANSA-Stack/SANSA-Examples/tree/master/sansa-examples-flink)
 Contains the SANSA Examples for [Apache Flink](http://flink.apache.org/).
+
+## Not able to build the project with maven then read this section:
```

Comment:
Please, remove the whole section above. We want to keep the repo clean from such instructions, most of the developers who start using SANSA are familiar with the maven installation instructions and this could be relevant for your environment settings only. In the case of the students attending our lab, we also provide some basic instructions on our Big Data lab here: https://github.com/SmartDataAnalytics/MA-INF-4223-DBDA-Lab/blob/master/labs/WorkSheet-1.md and maybe this is relevant to be added there, so feel free to suggest changes on the worksheet:).

Reply:
Hi Gezim,

Thanks for reviewing the PR. I see. Then I removed and cleaned README.md :)

Warmest regards,
Mohammad